plugins_tools/eid-viewer/OSX/support/ossl/GNUmakefile

```diff
@@ -1,53 +1,131 @@
-OSSL_VERSION=1.1.1i
-TP_DIR=../../../../../../ThirdParty
+# GNU Makefile for building ReSSL
+#
+LIBRESSL_VERSION=3.3.1
+TP_DIR=$(shell pwd)
```

Comment:
No.

This is outside of the git checkout directory for a reason. Please keep it that way.